Automattic/wp-calypso
# Fix/stepper setup form input field

#### Proposed Changes

The post setup form used within the stepper framework for the Link-in-bio, Newsletter, and Free onboarding flow has an issue where the input fields sometimes ignore the first key press if the fields are prefilled. This does not occur on the initial setup screen.

These changes aim to resolve that issue to improve user experience.

#### Testing Instructions



* Checkout this branch
* Create a new site or use an existing site (recommend creating a newsletter and replacing the URL to test the other flows)
* Navigate to the setup screen (confirm issue is not occurring on these screens) and post-setup screen for each flow and verify the issue has been fixed:

Newsletter post setup screen: `setup/newsletter-post-setup/newsletterPostSetup?siteSlug={SITESLUG}`
Link-in-bio post setup screen: `setup/link-in-bio-post-setup/linkInBioPostSetup?siteSlug={SITESLUG}`
Free flow post setup screen: `setup/free-post-setup/freePostSetup?siteSlug={SITESLUG}`

* Confirm the first key press is not ignored and that the form fields are prefilled when site data is available.
* Confirm issue is not occurring on initial setup screens

Newsletter setup screen: `setup/newsletter/newsletterSetup`
Link-in-bio setup screen: `setup/link-in-bio/linkInBioSetup`
Free flow setup screen: `setup/free/freeSetup`



#### Pre-merge Checklist





Related to #
